lib/hooks/docker-hub-automated-build/templates/default.html.haml

```diff
@@ -0,0 +1,4 @@
+%div
+  #{payload.push_data.pusher} pushed #{payload.repository.repo_name}:#{payload.push_data.tag}
+  %br
+  #{payload.repository.repo_url}
```

Comment:
I changed the template to link `repo_name` to the repository url, It looks nicer now. thanks!

![2017-06-05 18 41 06](https://cloud.githubusercontent.com/assets/167012/26779117/236ca51e-4a1f-11e7-9b84-d35a47b1e54a.png)